### DISCUSSION TITLE
Is there a readable and writable class?

### DISCUSSION THREAD

**Discussion by HaoXuan11** [rikyoz/bit7z#139] [Q&A] [answered]
Is there a readable and writable class?

**Comment by rikyoz** [comment, marked as answer]
Hi!
Unfortunately, bit7z doesn't provide a class that allows _both_ reading from and writing to an archive.
It only provides separate classes for these operations.
This was done mainly to simplify the code, but I'll evaluate whether to add such a class in a future version.

**Reply by HaoXuan11** [reply]
Thank you .
